src/cmd_processor.h

```diff
@@ -0,0 +1,22 @@
+#ifndef BLACKBOARD_CMDPROCESSOR_H
+#define BLACKBOARD_CMDPROCESSOR_H
+
+#include <memory>
+#include <string>
+
+namespace bb
+{
+    class CMDProcessor
+    {
+    public:
+        CMDProcessor();
+        virtual ~CMDProcessor();
```

Comment:
I think that a virtual destructor is not needed here

Reply:
removed! b09ab54